lib-ee/emqx_license/src/proto/emqx_license_proto_v2.erl

```diff
@@ -11,20 +11,15 @@
 -export([introduced_in/0]).
 
 -export([
-    remote_connection_counts/1,
-    save_and_backup_license/2
+    remote_connection_counts/1
 ]).
 
 -define(TIMEOUT, 500).
 -define(BACKUP_TIMEOUT, 15_000).
 
 introduced_in() ->
-    "5.0.5".
+    "e5.0.0".
```

Comment:
Shouldn't be the proto index file be updated?

Reply:
Maybe surprisingly, no alarms went of in the static checks. :thinking: 

Also, I think that [`introduced_in`](https://github.com/emqx/emqx/blob/982abd96bf199e82bee25f351797539f71031472/apps/emqx/src/bpapi/emqx_bpapi_trans.erl#L67) is currently purely informational, not being recorded anywhere.